jcli/src/jcli_app/mod.rs

```diff
@@ -52,6 +53,8 @@ pub enum JCliCommand {
     AutoCompletion(auto_completion::AutoCompletion),
     /// Utilities that perform specialized tasks
     Utils(utils::Utils),
+    /// Vote related operations
+    Votes(vote::Vote),
```

Comment:
why not put the key generation under `Key`?

Reply:
Actually yes, I didn't even think about it, I just wanted to keep all voting-related tools together. But I'm down for the key option too.